### ISSUE TITLE
Can't get the last cutscene on quest "Got it all"

### ISSUE THREAD

**Issue by Balrahn** [EdenServer/community#1564]
### Checklist

- [x ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ x] I have provided my client version in the details. (type /ver into your game window)

### Details

Version: 30181205_0

I can't remember when I started this quest but it was close to the release of Aht Urghan.
I followed the steps as stated in this guide on the wiki: https://ffxiclopedia.fandom.com/wiki/Got_It_All

I can't remember what I answered when the quest npc asked me questions.

I never got the seashell after zoning. Tried waiting until after Jp midnight and still nothing.
Now it's been maybe months and still I get nothing.

**Comment by Shoruto** [comment]
I saw a post regarding this that said they had to wait 3 days or until conquest resets to finally get the CS, 

I also saw on the quest if you change the date to era time, there are a good number of NPCs you have to talk to and several zoning,  I would suggest following the steps from the beginning one by one, this might allow whatever might have been missed to be corrected.  Then before the final step, wait till conquest resets then try talking to Tehf and see if you get the seashell.

**Comment by bkesecker** [comment]
@Balrahn if waiting longer fixes the issue, please close this GitHub issue.

**Comment by Balrahn** [comment]
So I did as Shoruto said. I managed to get a CS from Hajaom. None of the npc's did anything new after I got that CS. I've waited now for the next tally and none of the npc's are giving me anything new at all. Still stuck.

**Comment by Balrahn** [comment]
This is a little weird.. I did as Shoruto said and retraced my steps. Found out that I were missing a CS from Hajaom. I got the cs by speaking to him (I remember getting some error message, like there were some conflicting quests or something. Can't remember what it sais.). After that nothing happened. Couldn't talk to any npcs to progress. Waited for tally reset and still nothing. I then walked all up next to Hajaom and I got the same CS. I got a new CS with Tehf after this so it's probably working now.
